src/analyze/analyze-unit-files.c

```diff
@@ -25,7 +25,7 @@ int verb_unit_files(int argc, char *argv[], void *userdata) {
         if (r < 0)
                 return r;
 
-        r = unit_file_build_name_map(&lp, NULL, &unit_ids, &unit_names, NULL);
+        r = unit_file_build_name_map(&lp, NULL, &unit_ids, &unit_withdrawals, &unit_names, NULL);
```

Comment:
This would be quite some refactoring, as manager members are passed as arguments: https://github.com/systemd/systemd/blob/30e29edf4c0bb025aa7dc03c415b727fddf996ac/src/core/load-fragment.c#L6070-L6074